src/SamplesApp/SamplesApp.Shared/App.xaml

```diff
@@ -109,6 +112,19 @@
 						Value="LightBlue" />
 			</Style>
 
+			<Style x:Key="DefaultCommandBarStyle"
+					TargetType="CommandBar"
+					xamarin:BasedOn="{StaticResource NativeDefaultCommandBar}">
```

Comment:
Does the test work if you omit this `BasedOn` line? If so it's preferable, otherwise this will interfere with any tests, screenshot-diffing etc which was previously using the UWP CommandBar style.